tvb_framework/tvb/tests/framework/interfaces/web/controllers/simulator_controller_test.py

```diff
@@ -90,7 +92,8 @@ def test_index(self):
         assert not result_dict['errors'], 'Some errors were encountered!'
 
     def test_set_connectivity(self):
-        zip_path = path.join(path.dirname(tvb_data.connectivity.__file__), 'connectivity_66.zip')
+        #zip_path = path.join(path.dirname(tvb_data.connectivity.__file__), 'connectivity_66.zip')
+        zip_path = TVBZenodoDataset().fetch_data('connectivity_66.zip')
```

Comment:
In tests like this one, where multiple `fetch_data` are used, can't we keep a single instance `TVBZenodoDataset()` on the test class and reuse it? It seems that few operations are done in the constructor, and we might gain some time if we do it only once.